api/nodes/serializers.py

```diff
@@ -1225,6 +1235,13 @@ class DraftRegistrationSerializer(JSONAPISerializer):
         related_view_kwargs={'schema_id': '<registration_schema._id>'}
     )
 
```

Comment:
Was going to ask if these were filterable, but I don't see filterable fields. 

So probably out of scope to add but figured I'd bring up now in case there was a use case.